thunder/torch/__init__.py

```diff
@@ -1766,6 +1766,11 @@ def real(a):
     return clang.real(a)
 
 
+@torchsymbol(torch.imag, is_method=False)
```

Comment:
Isn't `imag` also a method? 

https://pytorch.org/docs/stable/generated/torch.Tensor.imag.html

Reply:
Thank you! After making the change in my latest commit, this works:
```python

In [1]: import torch
   ...: import thunder
   ...: 
   ...: def f(x):
   ...:     z = x.real
   ...:     return z
   ...: 
   ...: def g(x):
   ...:     z = torch.real(x)
   ...:     return z
   ...: 
   ...: jf = thunder.jit(f)
   ...: jg = thunder.jit(g)
   ...: 
   ...: x = torch.rand((2, 2), dtype=torch.complex128)
   ...: 
   ...: f(x)            # OK  
   ...: g(x)           # OK  
   ...: 
   ...: jf(x)           # OK
   ...: jg(x)
Out[1]: 
tensor([[0.5807, 0.8727],
        [0.1300, 0.8024]], dtype=torch.float64)
```